daemon/albatross_tls_endpoint.ml

```diff
@@ -166,7 +166,7 @@ let ip =
   Arg.(value & opt ip_c Ipaddr.(V6 V6.unspecified) & info [ "ip" ] ~doc)
 
 let inetd =
-  let doc = "Inetd mode" in
+  let doc = "Inetd mode. Be sure to use `--syslog` (or `--quiet` to disable logging)" in
```

Comment:
Is it perhaps worth it to check this is the case?

Reply:
Would c12a53f233d0389d549eaf7b84a2853a832870ba be fine in that regard? The issue is that you'll only see exit code 3... since the tls-endpoint binary is only ever called by inetd where stdout (and stderr I believe) are not available anymore...

So, actually I'm not sure whether it is worth to have this check in the code.

Reply:
Ah this opens another can of worms. If you pass invalid flags/arguments to albatross-tls-endpoint I think cmdliner will print out help messages?! I think to stderr. If so then that's as well problematic.

Reply:
Maybe another approach is to have a `albatross-tls-endpoint-inetd` that is a symlink to `albatross-tls-endpoint` and match on `Sys.argv.(0)` to determine if we're `-inetd` and then pass to `Cmdliner.Cmd.eval` null formatters `~help` and `~err` and set inetd mode.

Reply:
I wouldn't overengineer this. :) Given that for e.g. Linux and systemd we have the socket activation, the inetd mode is mostly a relict for our FreeBSD server :)